packages/flutter/lib/src/painting/alignment.dart

```diff
@@ -32,6 +32,33 @@ abstract class AlignmentGeometry {
   /// Creates a directional alignment, or [AlignmentDirectional].
   const factory AlignmentGeometry.directional(double start, double y) = AlignmentDirectional;
 
+  /// The top left corner.
+  static const AlignmentGeometry topLeft = Alignment.topLeft;
```

Comment:
Nit: Would it be useful to link these to their Alignment equivalent in the docs? Or maybe AlignmentGeometry shouldn't technically know about Alignment? Anyway, up to you.